ui/src/components/ExportDialog.tsx

```diff
@@ -186,16 +193,43 @@ export default function ExportDialog({ open, onClose }: Props) {
     );
   };
 
+  const renderPushToRegistryRadioButton = () => {
+    return (
+      <>
+        <FormControlLabel
+          value="push-registry"
+          control={<Radio />}
+          label="Registry"
+        />
+        {fromRadioValue === "push-registry" && (
+          <Stack pt={1} pb={2} pl={4} width="100%">
+            <Typography pb={2} variant="body2">
+              Push the volume content to a registry like DockerHub or GitHub
+              Container Registry.
+            </Typography>
+            <TextField
+              fullWidth
+              label="<registry>/<user>/<repo-name>:<tag>"
```

Comment:
Add front-end validation to ensure that the input contains _at least_ the user, e.g:

✅ registry/johhn/repo-name:0.0.1 (registry is optional, if not specified, DockerHub will be used by default)

✅john/repo-name:0.0.1

❌ repo-name:0.0.1

Reply:
added some validation 👍